pageserve/test_routes.py

```diff
@@ -32,9 +32,14 @@
 ERROR_BAD_TOKEN_STATUS = 400
 VALID_USER_AUTH_STATUS = 201
 
-EXAMPLE_USER_OBJECT = {
+AUTHORIZED_USER_OBJECT = {
     'user_id': 'app_user',
-    'name': 'app_user'}
+    'name': 'app_user',
+    'is_organizer': True}
```

Comment:
I've noticed that some user objects use `'edit_access'` and others use `'is_organizer'` - we should standardize to one or the other.

Reply:
Depending on how you did it for the users service, I think only some of my old pageserve code uses `'edit_access'` so it's probably best to replace that.

Reply:
I'll open a separate PR to convert everything to `'is_organizer'`.

Reply:
#68